lib/runners/objc.js

```diff
@@ -1,37 +1,82 @@
 var shovel = require('../shovel'),
     util = require('../util'),
+    temp = require('temp'),
     path = require('path'),
-    temp = require('temp');
+    exec = require('child_process').exec;
+
+function compile(args, cb) {
+    args.unshift('gcc', '`gnustep-config --objc-flags`');
+    exec(args.join(' '), cb);
+}
+
+function getCode(opts)
+{
+    const code = [];
+
+    if (opts.format === "default")
```

Comment:
The CLI uses the `format` param to indicate if the response should be JSON, so this conflicts.